CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Changed
```

Comment:
nit: well, it would be better to be a `### Added` section in my opinion.
Please feel free to go ahead and merge it after resolved, thanks

Reply:
🤦 resolved